Floofbot/Services/NicknameAlertService.cs

```diff
@@ -166,6 +169,11 @@ public async Task OnReactionAdded(Cacheable<IUserMessage, ulong> message, ISocke
                     }
                     alertMessageIdsDic.Remove(msg.Id);
                 }
+                else if (reaction.Emote.Name.Equals(NO_ACTION_EMOJI.Name))
```

Comment:
Actually, for the REMOVE option should we change the message to "I removed ..." rather than "I changed ..."?